README.md

```diff
@@ -41,6 +41,30 @@ compiled to WASM.
 
 Arbitrum One successfully migrated from the Classic Arbitrum stack onto Nitro on 8/31/22. (See [state migration](https://developer.arbitrum.io/migration/state-migration) and [dapp migration](https://developer.arbitrum.io/migration/dapp_migration) for more info).
 
+## Running E2E Tests
+
+### Prerequisites
+- Nix package manager
+- Docker daemon running
+
+### Build Steps
+1. For MacOS Users Only:
+```bash
+bash ./scripts/build-wasm-on-macos-with-nix
+```
+2. Enter development environment:
+```bash
+nix develop
+make build
```

Comment:
I think `make build` and `make build-replay-env` should be the 3rd step

Reply:
Oh yes true, I'll update that